src/main/java/com/zufar/icedlatte/favorite/entity/FavoriteList.java

```diff
@@ -0,0 +1,46 @@
+package com.zufar.icedlatte.favorite.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "favorite_list")
+public class FavoriteList {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @Column(name = "user_id", nullable = false)
```

Comment:
Actually, it is default Hibernate behavior to only load the foreign key instead of the User object if the FetchType is LAZY. This is why there are proxies to the objects to be loaded when you specify LAZY FetchType. The foreign key is not visible directly, but it is of course the key of the object at the "one" end of the OneToMany relationship.